packages/ingest/src/lock.ts

```diff
@@ -17,10 +21,18 @@ const lockPrefix = 'lock';
 * @param {Array} locks - The list of locks in the bucket
 * @returns {integer} - Number of locks remaining in bucket
 **/
-async function checkOldLocks(bucket, locks = []) {
+export async function checkOldLocks(
```

Comment:
`checkOldLocks` is horribly named, since it returns a list of locks that are **not** expired, which is the opposite of what I expect based on the name. But I digress

Reply:
Yeah, this entire file needs to go and be replaced by the Dynamo semaphores.